% metagenomics - raw sequencing data
% genomic dark matter - cant classify everything
% structural (what bacteira are there) and functional (what does the community do)

%%% CHAPLAYOUT %%% 

%%% IDEAS %%% 
microbiome is all the microognisms in an environment
machine learning
work out the key bacteria that make people healthy
datasets are large
sequencing costs diminish (hmp), analysis is the costly
resources allows to upload (mgnify)
snakemake - allows for the design of schema and monitoring of resources during jobs.
	the CWL is the common workflow language that does roughly the same thing

proprietary vs open source (democritization)
amplicon vs shotgun sequencing
clustering first vs assignment first pipelines
metrics for pipelines
pipeline standards
each chapter as a step in the pipeline (QC filtering, classifying, annotation etc)

%%% DATABASE %%% 
EBI

%%% SOFTWARE %%% 
MetaWRAP - 170
MetAMOS - 167
ATLAS - Snakemake workflow - 6 - 2020
MGnify (from EBI)- 42 - 2020
METEOR - 7 - 2010
Galaxy (ASaiM) - server has 3656C - 2010
fastqc

from 2017 review - 
BMP, mothur, QIIME, CLARK, Kraken, One Codex

Pipelines::
SPAdes
BayesHammer
MismatchCorrector
Whole pipeline
MetaSUB Core Analysis Pipeline (CAP)
metaphlan3

list of EBI pipelines:
https://www.ebi.ac.uk/metagenomics/pipelines/3.0
